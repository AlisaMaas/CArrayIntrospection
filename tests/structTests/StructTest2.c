/**
* Access a struct with a recursive element 
* and no other structs around.
**/

struct foo {
	int a;
	double b;
	struct foo *c;
};

struct bar {
	struct foo first;
	int x;
	double y;
	struct bar *z;
};

void accessFoo(struct foo f) {
	f.a = 32;
	f.b = 1.2;
	f.c = &f;
}