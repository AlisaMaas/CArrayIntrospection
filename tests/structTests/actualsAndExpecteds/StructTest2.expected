struct.foo:
	element 0 accessed
	element 1 accessed
	element 2 accessed
